kolibri/core/assets/test/vue.progress-bar.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-env mocha */
+// The following two rules are disabled so that we can use anonymous functions with mocha
+// This allows the test instance to be properly referenced with `this`
+/* eslint prefer-arrow-callback: "off", func-names: "off" */
+
+'use strict';
+
+const Vue = require('vue-test');
+const progressBar = require('../src/vue/progress-bar');
+const ProgressBarComponent = Vue.extend(progressBar);
+const assert = require('assert');
+
+describe('progressBar Component', function () {
+  describe('computed properties', function () {
+    describe('percent', function () {
+      beforeEach(function () {
+        this.vm = new ProgressBarComponent({
+          propsData: {
+            progress: 0,
+          },
+        }).$mount();
+      });
+      it('should give 0 percent for progress of < 0', function () {
```

Comment:
@aronasorman @rtibbles  Is there a rule of thumb in deciding what to test for a component?